src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -71,4 +71,10 @@ private Line findLineById(Long id) {
 		return lineRepository.findById(id)
 			.orElseThrow(() -> new IllegalArgumentException("노선이 존재하지 않습니다."));
 	}
+
+	public void deleteStationByIdInLine(Long lineId, Long stationId) {
+		Line line = findLineById(lineId);
+		Station station = stationService.findStationById(stationId);
+		line.deleteStationInSections(station);
```

Comment:
```suggestion
		line.deleteStation(station);
```
이정도면 충분히 의미 전달이 되지 않을까요?
내부 구현 상황을 메서드명에 표현할 필요는 없습니다.